### PR TITLE
follow up PR to use new inputs in profile

### PR DESCRIPTION
**Pull request by rohit1290**
- Ref https://github.com/Elgg/Elgg/pull/12644#issuecomment-511794301 - added new input type for profile
- fix theme_sandbox type for datetime-local

### CODE REVIEW

File: mod/profile/languages/en.php [hunk 1]
```diff
@@ -29,8 +29,13 @@
 	'profile:field:tags' => 'Tags',
 	'profile:field:url' => 'Web address',
 	'profile:field:email' => 'Email address',
+	'profile:field:telephone' => 'Telephone',
```
Comment (jdalsem) [inline mod/profile/languages/en.php]: make it `profile:field:tel` so it is in line with the input types

**Comment by jdalsem** [comment]
can you reword the `fix` commit to a chore? This needs to be done because a fix commit will end up in the release notes by our automated release process. The problem this commit fixes is not released yet, so no need to have it mentioned in the release notes

**Comment by jdalsem** [comment]
maybe also reword your feat commit to: `feat(profile): new input types for custom profile fields`... it is always a challenge to come up with a good description of your commit. It should be understandable if you read it in the release notes

**Comment by rohit1290** [comment]
@jdalsem - done

**Comment by jdalsem** [comment]
Thanks! that was quick :)

**Pull request merged**